terraform/20-app/waf.public-api.tf

```diff
@@ -86,14 +86,6 @@ locals {
         priority = 3
         name     = "AWSManagedRulesAmazonIpReputationList"
       },
-      {
-        priority = 4
-        name     = "AWSManagedRulesLinuxRuleSet"
-      },
-      {
-        priority = 5
-        name     = "AWSManagedRulesUnixRuleSet"
-      }
```

Comment:
Are you sure you want to remove these?

Reply:
Yeah they were never counted. Happy to keep them if you want